pkg/config/config_template.yaml

```diff
@@ -1708,6 +1710,28 @@ api_key:
   ##
   # check_max_events_per_run: 100
 {{ end -}}
+
+{{- if .SBOM }}
+## @param sbom - custom object - optional
+## Enter specific configuration for the Cloud Security Management Vulnerability Management feature
+# sbom:
+  ## @param enabled - boolean - optional - default: false
+  ## set to true to enable Cloud Security Management Vulnerability Management
+  # enabled: false
+
+  ## uncomment the sections below to enable where the vulnerability scanning is done
+
+  ## @param host
+  ## enable the host-level check
+  # host:
+  #  enabled: false
```

Comment:
## @param enabled - boolean - optional - default: false
## set to true to enable Infrastructure Vulnerabiltilies

Reply:
thanks @lebauce added in your suggestions